tooling/nargo/src/foreign_calls/rpc.rs

```diff
@@ -19,6 +19,8 @@ pub struct RPCForeignCallExecutor {
     id: u64,
     /// JSON RPC client to resolve foreign calls
     external_resolver: HttpClient,
+    /// External resolver target. We are keeping it to be able to restart httpClient if necessary
```

Comment:
Should we add a link to the "TXE server sometimes dies" issue, or does this refer to a different thing?